src/api/app/models/event/added_user_to_group.rb

```diff
@@ -0,0 +1,11 @@
+module Event
+  class AddedUserToGroup < Base
+    self.message_bus_routing_key = 'groups_user.create'
+    self.description = 'Added user to group'
+    payload_keys :group, :user
```

Comment:
I'm also missing `who` did that change here...

Reply:
Aha.  I didn't see any copy for the messages;  this was basically a placeholder at the time.